### ISSUE TITLE
Wrong NP_NULL_ON_SOME_PATH warning if Preconditions check contains line break 

### ISSUE THREAD

**Issue by BrainStone** [spotbugs/spotbugs#462]
I came across this weird error. When the null check is on a separate line than the corresponding checking method I get a `NP_NULL_ON_SOME_PATH` warning.

Working (no warning): 
```Java
import java.util.Map;
import com.google.common.base.Preconditions;

public class TestWorking {
  final Object initialState;

  public TestWorking(Map<Object, ?> states) {
    Preconditions.checkState((states != null) && !states.isEmpty());

    this.initialState = states.keySet().iterator().next();
  }
}
```

Not working (warning):
```Java
import java.util.Map;
import com.google.common.base.Preconditions;

public class Test {
  final Object initialState;

  public Test(Map<Object, ?> states) {
    Preconditions.checkState(
        (states != null) && !states.isEmpty());

    this.initialState = states.keySet().iterator().next();
  }
}
```

**Comment by ctubbsii** [comment]
Saw this when my project changed from line length of 160 to 100, and formatted. :frowning_face:

**Comment by ctubbsii** [comment]
Haven't seen this in awhile, but curious if anybody has noticed if it is fixed or not.
